hardware/Layout - desing- guide/Layout-Desing-Guide.md

```diff
@@ -0,0 +1,464 @@
+
+# EFR32 Layout Design Guide
+
+The purpose of this material is to help users design PCBs
+for the EFR32 Wireless Gecko Portfolio using design practices
+that allow for good RF performance.
+
+PCB layout requirements are described in the application note, [AN928.1: EFR32 Series 1 Layout Design Guide](https://www.silabs.com/documents/public/application-notes/an928.1-efr32-series1-layout-design-guide.pdf),  and the layout design process for EFR32 Series 1 Modules is discussed in [the datasheet of each module](https://www.silabs.com/support/resources.ct-data-sheets.ct-miscellaneous.p-wireless_bluetooth-low-energy_blue-gecko-bluetooth-low-energy-modules.p-wireless_bluetooth-low-energy_blue-gecko-bluetooth-low-energy-modules_bgm11s12f256ga-v2.p-wireless_bluetooth-low-energy_efr32bg1-series-1-modules.p-wireless_bluetooth-low-energy_efr32bg13-series-1-modules.sort=2,asc) and  beside that EFR32 reference design files are available in Simplicity Studio.
+
+
+## Why is the proper layout design important? 
```

Comment:
It would be good to have number bullet. We can start with HEAD 1 "#"  instead of HEAD 2 "##" here. No need to have HEAD 1 "#EFR32 Layout Design Guide" at beginning of the article. The GitHub wiki will take the file name as the default title name

Reply:
Done